core/src/main/resources/lib/form/radio.jelly

```diff
@@ -43,9 +43,14 @@ THE SOFTWARE.
       toggles the radio.
     </st:attribute>
   </st:documentation>
-  <input type="radio" name="${attrs.name}" onclick="${attrs.onclick}" id="${attrs.id}" value="${attrs.value}"
-        checked="${attrs.checked?'true':null}" disabled="${readOnlyMode?'true':null}" />
-  <j:if test="${attrs.title!=null}">
-    <label class="attach-previous">${attrs.title}</label>
-  </j:if>
+  <j:set var="itemId" value="${h.generateItemId(id)}" />
```

Comment:
isn't this a breaking change to the component? you were allowed to set the ID before and people could be relying on this.

I think it should use the ID if present otherwise generate one?

Reply:
You're right - I've updated it so that it'll only generate an ID if one isn't provided.